pkg/cache/cache.go

```diff
@@ -688,6 +688,25 @@ func (c *Cache) Usage(cqObj *kueue.ClusterQueue) (*ClusterQueueUsageStats, error
 	return stats, nil
 }
 
+type CohortUsageStats struct {
+	WeightedShare int64
+}
+
+func (c *Cache) CohortUsage(cohortObj *kueuealpha.Cohort) (*CohortUsageStats, error) {
+	c.RLock()
+	defer c.RUnlock()
+
+	cohort := c.hm.Cohort(kueue.CohortReference(cohortObj.Name))
```

Comment:
is there any chance this will return nil? if so, check and return err. If not, remove error from return value

Reply:
Done